source/src/SadWalker.kt

```diff
@@ -0,0 +1,7 @@
+class SadWalker(private val velocity: Int = 5) : IWalker {
+    override fun walk() {
+        val speed: Speed = Speed(velocity)
+        println("I`m a sad walker and my ")
+        speed.showSpeedConsole()
+    }
```

Comment:
So, now i can see that it is not comfortable design of this program.
You need make class instance, feed the variables there, and call method's of this object.
Why not just use a static method that gets the speed and prints it out?